### PR TITLE
feat: add method to index certain errors on Sentry - ADMT-541

### PR DESCRIPTION
**Pull request by marcelovicentegc**
We need to index certain types of errors that are not yet indexed. To do this, we need to transform error objects into a "taggable" structure (such as key:value pairs). This PR adds a method to recursively extract keys from objects and adapt them to the our convention and signature expected on Sentry.

For more details, let's talk on Slack :hugs: 

Tests:

image

Depends on https://github.com/vtex/render-server/pull/801 to take effect.

### CODE REVIEW

**Review by vsseixaso — APPROVED** [review]
LGTM!
Very good cases validated in the tests!

File: react/o11y/extractExtra.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          // Sentry's limit of 200 characters per value
+          if (value.length >= 200) {
+            const chunks = []
+            for (let chunk = 0; chunk < value.length; chunk += 199) {
+              chunks.push(value.substr(chunk, 199))
+            }
+
+            for (let chunk = 0; chunk < chunks.length; chunk++) {
+              extra[
+                `admin_extra_level_${iterationNumber}_` + key + '_' + chunk
```
Comment (vmourac-vtex) [inline react/o11y/extractExtra.ts]: @marcelovicentegc acho que essa regra de formação de chaves tem potencial pra bater no [limite do tamanho da key](https://docs.sentry.io/platforms/javascript/guides/node/enriching-events/tags/). A chave proposta já tem um min de 22 chars. Uma `key`com 10 chars já bate no limite.

> Tag keys have a maximum length of 32 characters and can contain only letters (a-zA-Z), numbers (0-9), underscores (_), periods (.), colons (:), and dashes (-).

Pra diminuir um pouco o risco, o que acha de algo assim:
`crop_${iterationNumber}_${key}_${chunk}`
ou 
`${key}_crop_${iterationNumber}_${chunk}`

**Review by vmourac-vtex — CHANGES_REQUESTED** [review]
@marcelovicentegc pedi uma revisão pois parece q temos um risco de estourar tamanho da Key ao gerar iterations/crops

File: react/o11y/extractExtra.ts (hunk 1 above)
Comment (marcelovicentegc) [reply react/o11y/extractExtra.ts]: Perfeita anotação @vmourac-vtex ! Modifiquei a geração da chave para encurtá-las. Me diz o que achou :hugs: !

**Review by vmourac-vtex — APPROVED** [review]
Rapaz, tu trabalha rápido hein! Ficou ótimo!

**Pull request merged**